### ISSUE TITLE
:focus missing 

### ISSUE THREAD

**Issue by csps-snap** [DIS-SIN/Busrides#83]
WCAG 2.4.7 problem: focus is lost
(for wcag reference)
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

![image](https://user-images.githubusercontent.com/41083520/138280013-50be0a46-f886-4243-befd-301cfa648ece.png)

**Comment by csps-snap** [comment]
also all cards with the little faces in them have focus lost problems
![image](https://user-images.githubusercontent.com/41083520/138280396-1a7450aa-499b-48ee-ab2a-1d5a1924d2fe.png)

**Comment by csps-snap** [comment]
this guy feels sad he doesn'T focus either
![image](https://user-images.githubusercontent.com/41083520/138280679-c8712121-bb41-4525-a781-1249be810488.png)
